triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceConfig.scala

```diff
@@ -89,10 +91,12 @@ object ServiceConfig {
       .action((f, c) => c.copy(darPath = Some(Paths.get(f))))
       .text("Path to the dar file containing the trigger.")
 
-    opt[Int]("http-port")
-      .optional()
-      .action((t, c) => c.copy(httpPort = t))
-      .text(s"Optional HTTP port. Defaults to ${DefaultHttpPort}.")
+    cliopts.Http.serverParse(this, serviceName = "Trigger")(
+      address = (f, c) => c.copy(address = f(c.address)),
+      httpPort = (f, c) => c.copy(httpPort = f(c.httpPort)),
+      defaultHttpPort = Some(DefaultHttpPort),
+      portFile = None,
```

Comment:
Side note: We should add a port file option but let’s keep that for a later PR. I’ll add an issue.